Sources/NIOFileSystem/FileSystemError+Syscall.swift

```diff
@@ -1016,6 +1016,25 @@ extension FileSystemError {
                 Can't copy file from '\(sourcePath)' to '\(destinationPath)', the destination \
                 path already exists.
                 """
+        case .fileExists:
```

Comment:
Can you update the error tests? They check that `errno`s get mapped to an appropriate file system error code.